### PR TITLE
button logic and UI adjustments

### PR DESCRIPTION
**Pull request by antontsitovich**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -28,13 +28,14 @@ const ContentBox = styled(Box)({
 const PositionedContent = styled(Box)({
   position: "absolute",
   width: "100%",
-  top: -141,
+  top: -133,
   left: 0,
 });
```
Comment (shaharyakir) [inline src/App.tsx:33]: absolute positioning will be hard for us to reason about and maintain, what can we do to refactor it? (if there are other areas in the app the use that let's refactor them too please)

File: src/components/dataBlock.styled.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { CenteringBox, DataBox } from "./common.styled";
 import { Box, Typography } from "@mui/material";
 
 const DataFlexibleBox = styled(DataBox)({
-  maxWidth: "49%",
+  maxWidth: "49.2%",
```
Comment (shaharyakir) [inline src/components/dataBlock.styled.tsx:6]: what does this fix? did we verify that the fix works responsively across different resolutions?

File: src/App.tsx (hunk 1 above)
Comment (shaharyakir) [reply src/App.tsx:33]: we can do it in a separate PR btw

File: src/components/dataBlock.styled.tsx (hunk 2 above)
Comment (antongotech) [reply src/components/dataBlock.styled.tsx:6]: combined with gap: 20px on upper level, it is used to always make gap between blocks to be 20px

File: src/App.tsx (hunk 1 above)
Comment (antongotech) [reply src/App.tsx:33]: Screen Shot 2022-11-23 at 12 49 45
Absolute position is used only in one place and this is the way to handle layout we have for the search bar and containers that overflow it

**Pull request merged**